packages/compass-import-export/src/utils/csv.ts

```diff
@@ -37,9 +37,29 @@ export type CSVParsableFieldType =
   | 'timestamp'
   | 'decimal'
   | 'regex'
-  | 'number'
+  | 'number' // like 'mixed', but for use when everything is an int, long or double.
   | 'mixed';
 
+export const CSVFieldTypeLabels: Record<CSVParsableFieldType, string> = {
```

Comment:
We might want to think through this order since it is the order that the options in the field type selects will appear in and right now it is essentially arbitrary. But it is easy enough to change in a PR later once people had a chance to play with it.